MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -800,4 +796,29 @@ - (void)testDismissEmbeddedSafariWhenDisabling {
                              }];
 }
 
+- (void)testShowDistributeDisabledAlert {
```

Comment:
@clpolet true. I'll look into this but we might need to actually do some nasty hacks to display UIAlertController behavior.